velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -88,12 +90,47 @@ SubstraitVeloxExprConverter::toVeloxExpr(
       return std::make_shared<core::ConstantTypedExpr>(
           veloxType, variant::null(veloxType->kind()));
     }
+    case ::substrait::Expression_Literal::LiteralTypeCase::kString:
```

Comment:
Would you add a test for string literal?

Reply:
I moved this code to https://github.com/facebookincubator/velox/pull/2856.